aggregator/src/app/BundleRouter.ts

```diff
@@ -24,14 +24,29 @@ export default function BundleRouter(bundleService: BundleService) {
   router.get(
     "bundleReceipt/:hash",
     async (ctx) => {
+      const pendingBundle = await bundleService.lookupBundle(ctx.params.hash!);
+      if (pendingBundle) {
+        ctx.response.status = 202;
+        ctx.response.body = {
+          status: "pending",
+          submitError: pendingBundle.submitError,
```

Comment:
I believe after 5 tries the bundle will be in a permanent failed state. Is there a way to communicate that here? That may be for another issue to address.

Reply:
Follow up seems appropriate, thanks :+1: